releases.md

```diff
@@ -29,14 +30,22 @@ As of 2018-11-16, PELUX 3.0 has been released! Documentation can be found
 * Neptune 3 UI
 
 ### Known issues
-* QtWebengine does not work in QtWayland on the Raspberry Pi 3. The browser app
-  is therefore not functioning as intended.
 * When running the QtAS versions of the platform, there are no ttys allocated on
   the terminal. To get a terminal, ssh to the target. If you don't know the IP
-  address of your target, use arp-scan to search for it on your network.
-* There is going to be minor hickups in the PELUX Development Handbook that
-  we have missed even though we've proofread them. Please point them out and we
-  will fix them!
+  address of your target, use arp-scan to search for it on your network. You can
+  also select the Qt button at the bottom right corner of the home screen and
+  scroll down to see the IP address.
+* Rasperry Pi 3 (Qt Auto Neptune variant):
+  - There are widgets and applications which are not functional. A workaround
+    is to force the UI into single process mode. Add argument
+    *--force-single-process* to neptune3-ui in the unit file for the neptune
+    service. You can either edit source file
+    *sources/meta-pelux/layers/b2qt/recipes-qt/automotive/neptune3-ui/neptune.service*
+    before building the image, or login to target via ssh and edit file
+    */lib/systemd/system/neptune.service* followed by a reboot.
+  - The Neptune 3 UI is not running as smooth as the former Neptune UI on
+    Rasperry Pi 3, but on NUC it's not an issue.
```

Comment:
Do we want to write "RPi 3B+" here also?

Reply:
it's a problem on the non-plus variant too, so likely all rasperry pis